src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -54,9 +54,10 @@ def create_ramp_fit_class(model, dqflags=None, suppress_one_group=False):
     ramp_data = ramp_fit_class.RampData()
 
     if isinstance(model.data, u.Quantity):
-        ramp_data.set_arrays(model.data.value, model.err.value, model.groupdq, model.pixeldq)
+        ramp_data.set_arrays(model.data.value, model.err.value, model.groupdq,
+                             model.pixeldq, model.average_dark_current)
     else:
-        ramp_data.set_arrays(model.data, model.err, model.groupdq, model.pixeldq)
+        ramp_data.set_arrays(model.data, model.err, model.groupdq, model.pixeldq, model.average_dark_current)
```

Comment:
Is `avg_dark_current` an array?  I thought it was a single value, which means it should passed to the `RampData` class using the `set_meta` method.

Reply:
I just read through the tests and see it is an array.  This PR is fine.